server/routes/induction/create/hopingToWorkOnReleaseCreateController.test.ts

```diff
@@ -5,19 +5,15 @@ import type { HopingToWorkOnReleaseForm } from 'inductionForms'
 import HopingToWorkOnReleaseCreateController from './hopingToWorkOnReleaseCreateController'
 import aValidPrisonerSummary from '../../../testsupport/prisonerSummaryTestDataBuilder'
 import HopingToGetWorkValue from '../../../enums/hopingToGetWorkValue'
-import validateHopingToWorkOnReleaseForm from '../../validators/induction/hopingToWorkOnReleaseFormValidator'
 import YesNoValue from '../../../enums/yesNoValue'
 
-jest.mock('../../validators/induction/hopingToWorkOnReleaseFormValidator')
-
 describe('hopingToWorkOnReleaseCreateController', () => {
-  const mockedFormValidator = validateHopingToWorkOnReleaseForm as jest.MockedFunction<
```

Comment:
I've taken the steer from your PR and discussion yesterday @leejacobson-moj re: whether we need to mock all of the dependencies ....

I'm not 100% convinced its right to use the real implementation in all cases, but I can certainly see the case for using the real validators to make the tests simpler

For all the controller tests I've had to touch in this PR for other reasons (and there are quite a few of them!), I've removed the mocking of the validator and refactored it to use the real implementation.

I will happily concede the tests are simpler in this respect 😁 👍